tests/foreman/api/test_host.py

```diff
@@ -266,6 +266,36 @@ def test_positive_create_inherit_lce_cv(self):
             hostgroup.content_view.id
         )
 
+    @tier2
+    def test_positive_create_with_inherited_params(self):
+        """Create a new Host in organization and location with parameters
+
+        :BZ: 1287223
+
+        :id: 5e17e968-7fe2-4e5b-90ca-ae66f4e5307a
+
+        :expectedresults: Host has inherited parameters from organization and
+            location
+
+        :CaseImportance: High
+        """
+        org = entities.Organization().create()
+        org_param = entities.Parameter(organization=org).create()
+        loc = entities.Location().create()
+        loc_param = entities.Parameter(location=loc).create()
+        host = entities.Host(
+            location=loc,
+            organization=org,
+        ).create()
+        self.assertEqual(
+            {org_param.name, loc_param.name},
+            {param['name'] for param in host.all_parameters}
+        )
+        self.assertEqual(
+            {org_param.value, loc_param.value},
+            {param['value'] for param in host.all_parameters}
+        )
+
```

Comment:
think more accurate assert
```python
all_parameters = {param['name']: param['value']  param in host.all_parameters}
org_loc_params = {org_param.name: org_param.value, loc_param.name: loc_param.value}
self.assertEqual(all_parameters, org_loc_params)
```
what do you think?

Reply:
if all_params is a dict
```python
org_loc_params = {org_param.name: org_param.value, loc_param.name: loc_param.value}
self.assertEqual(host.all_parameters, org_loc_params)

```

Reply:
I was thinking about that and decided to keep things simple. But since you've noticed that too, i should probably update assertions, thanks for suggestion 🙂

Reply:
Hm.. If not just param name but param value is duplicated too, sets won't show us that. We should probably check the `len()` as well, since list won't work for us as order can be easily damaged.